addons/better-quoter/addon.json

```diff
@@ -14,5 +18,7 @@
     }
   ],
   "tags": ["forums", "community"],
-  "l10n": true
+  "l10n": true,
+  "warning": "Scratchblocks tags are not supported, and the bbcode may not be 100% accurate. To get fully accurate quotes, quote as normal and delete any parts that you don't need."
```

Comment:
this was updated, use the system according to https://github.com/ScratchAddons/ScratchAddons/pull/957#issue-533087250

Reply:
and don't do warning, do notice.